packages/web-react/src/types/pagination.ts

```diff
@@ -44,3 +48,21 @@ export type SpiritPaginationLinkProps<E extends ElementType = 'a'> = PaginationL
 export type SpiritPaginationButtonLinkProps<E extends ElementType = 'a'> = PaginationButtonLinkProps<E>;
 
 export type SpiritPaginationLinkPreviousNextProps<E extends ElementType = 'a'> = PaginationLinkPreviousNextProps<E>;
+
+export interface UncontrolledPaginationProps {
+  accessibilityLabelNext?: string;
+  accessibilityLabelPrevious?: string;
+  defaultPage: number;
+  visiblePages?: number;
+  onChange: (pageNumber: number) => void;
+  totalPages: number;
```

Comment:
in the api docs nothing is required, but here defaultPage, onChange and totalPages are required